src/main/osd/osd_elements.c

```diff
@@ -992,6 +993,21 @@ static void osdElementFlymode(osdElementParms_t *element)
     }
 }
 
+static void osdElementReadyMode(osdElementParms_t *element)
+{
+    if (IS_RC_MODE_ACTIVE(BOXREADY) && !isFlying) {
+        if (calculateThrottlePercent() < 1) {
+            strcpy(element->buff, "READY");
+        } else {
+            isFlying = true;
+        }
+    }
+
+    if (!ARMING_FLAG(ARMED) && isFlying) {
+        isFlying = false;
+    }
```

Comment:
What about this:
```suggestion
    if (IS_RC_MODE_ACTIVE(BOXREADY) && !isFlying && calculateThrottlePercent() < 1) {
        strcpy(element->buff, "READY");
    }
    if (IS_RC_MODE_ACTIVE(BOXREADY)  && ARMING_FLAG(ARMED)) 
        if (calculateThrottlePercent() > 30) {
           isFlying = true;
        }
    } else {
        isFlying = false;
    }
```

- READY is displayed before start when BOX is enabled and throttle is zero
- - it is possible to hide READY by raising throttle a bit
- 30% throttle after arming will disable text - this threshold may be tuned a bit, opinions? 
- it is possible to restore READY both by disarming and (momentarily) disabling BOXREADY

Reply:
I like the momentary disable of ready and should definitely be considered.

I don't see the benefit of the 30% if they are flying let's remove ready